CHANGELOG.rst

```diff
@@ -4,13 +4,30 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
-1.0.0
------
+1.0.1 (2020-06-07)
+------------------
+
+**Added**
+
+* More units for the service products
+
+**Fixed**
+
+**Dependencies**
+
+* Bump data model lib 2.7.0 -> 2.8.0 and therefore adds more units for service products
+
+**Deprecated**
+
+1.0.0 (2020-06-04)
+------------------
 
 **Added**
 
 * Only list overhead costs of productGroup items present in offer in Offer PDF  (`#643 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/643>`_)
 
+* A person can now be updated if only the associated academic title is changed (`#567 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/567>`_)
+
```

Comment:
should not be part of 1.0.0 since that is not in our release

Reply:
It should be part of 1.1.0

Reply:
It seems this was already part of the last release, but I still dont get why this is brought up here again